File: common/src/main/java/com/imcys/bilibilias/common/base/adapter/AsJzvdStdAdapter.java
package com.imcys.bilibilias.common.base.adapter;

import androidx.databinding.BindingAdapter;

import com.bumptech.glide.Glide;
import com.imcys.bilibilias.common.base.view.AsJzvdStd;

import coil.Coil;

/**
 * @author:imcys
 * @create: 2022-11-18 17:39
 * @Description: 饺子播放器绑定器
 */
public class AsJzvdStdAdapter {

    @BindingAdapter({"android:imageUrl"})
    public static void loadImage(AsJzvdStd asJzvdStd, String url) {
        asJzvdStd.setPosterImageUrl(url);
        Glide.with(asJzvdStd.getContext())
                .load(url)
                .into(asJzvdStd.posterImageView);
    }
}

File: common/src/main/kotlin/com/imcys/bilibilias/common/base/adapter/AsJzvdStdAdapter.kt
package com.imcys.bilibilias.common.base.adapter

import androidx.databinding.BindingAdapter
import com.bumptech.glide.Glide
import com.imcys.bilibilias.common.base.view.AsJzvdStd

/**
 * @author:imcys
 * @create: 2022-11-18 17:39
 * @Description: 饺子播放器绑定器
 */
object AsJzvdStdAdapter {
    @BindingAdapter("android:imageUrl")
    fun loadImage(asJzvdStd: AsJzvdStd, url: String?) {
        asJzvdStd.posterImageUrl = url
        Glide.with(asJzvdStd.context)
            .load(url)
            .into(asJzvdStd.posterImageView)
    }
}
